setup.cfg

```diff
@@ -1,9 +1,7 @@
 [flake8]
 # References:
 # https://flake8.readthedocs.io/en/latest/user/configuration.html
-# https://flake8.readthedocs.io/en/latest/user/error-codes.html
-
-# Note: there cannot be spaces after comma's here
```

Comment:
This note is no longer accurate. Spaces are now allowed after commas.

Reply:
Spaces _are_ currently being used after commas.
https://github.com/dask/dask/blob/a4c571475b1c643dedf99bc773c2564bf3fcf977/setup.cfg#L6-L11